package/main/daml/Daml.Finance.Interface.Types.Date/daml.yaml

```diff
@@ -4,9 +4,11 @@
 sdk-version: 2.5.0
 name: daml-finance-interface-types-date
 source: daml
-version: 1.0.0
+version: 1.0.1
 dependencies:
   - daml-prim
   - daml-stdlib
+data-dependencies: null
```

Comment:
Not sure this is a desired output?

Reply:
Nut sure either, I suppose that is from `packell`. @brianweir-da , do you know? Should I just delete that line manually?

Reply:
I think its appeared like that because "data-dependencies:" was not originally defined in this file. You can delete this for now but its something that we'd need to look at in `packell` (there's not many packages that don't have a dependency in daml-finance). Can you create a issue for this?

Reply:
Thanks. I have removed the line now.
Here is the issue: https://github.com/digital-asset/daml-finance/issues/574